realm/realm-library/src/androidTest/java/io/realm/CollectionTests.java

```diff
@@ -34,7 +34,9 @@
 
     // Enumerates all known collection classes from the Realm API.
     protected enum CollectionClass {
-        MANAGED_REALMLIST, UNMANAGED_REALMLIST, REALMRESULTS
+        MANAGED_REALMLIST, UNMANAGED_REALMLIST, REALMRESULTS,
+        REALMRESULTSSNAPSHOT_RESULTS_BASE, REALMRESULTSSNAPSHOT_LIST_BASE
```

Comment:
`REALMRESULTS_SNAPSHOT_RESULTS_BASE` and `REALMRESULTS_SNAPSHOT_LIST_BASE` maybe